packer/packer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "variables": {
+    "aws_access_key": "Your AWS access key",
```

Comment:
Environment variables here are better, to avoid pushing secrets to the repo accidentally.